### PR TITLE
Fixed incorrect sample.

### PR DESCRIPTION
**Pull request by ericzhang6222**
Fixed incorrect sample.

### CODE REVIEW

File: docs/lang-spec.md [hunk 1]
```diff
@@ -790,8 +790,21 @@ Views are sysl's functions; we can use them in the transformation language, see
...
+    TypeInt32
+  
+  !alias TypeString:
+    string
+  
+  !alias TypeInt32:
+    int32
+
+  !type User:
+    id <: TypeStringOrInt32
```
Comment (ChloePlanet) [inline docs/lang-spec.md:805]: Not only `!alias` can be used in `!union`, so does `!type`
```suggestion
  !type User:
    id <: TypeStringOrInt32
    
  !union TypeStringOrInt32:
    TypeString
    TypeInt32
  
  !alias TypeString:
    string
  
  !type TypeInt32:
    id int32
```

**Comment by ChloePlanet** [comment]
Fixed https://github.com/anz-bank/sysl/issues/172.

File: docs/lang-spec.md (hunk 1 above)
Comment (ericzhang6222) [reply docs/lang-spec.md:805]: This sample is representing the usage of `!union`, it can add `!alias` for `!type ` individually.

Comment (ChloePlanet) [reply docs/lang-spec.md:805]: This sample only shows alias not type

**Pull request merged**